bigframes/ml/llm.py

```diff
@@ -90,7 +90,7 @@ def _create_bqml_model(self):
         )
         if self.model_name == "text-bison":
             options = {
-                "remote_service_type": _REMOTE_TEXT_GENERATOR_MODEL_CODE,
+                "endpoint": _REMOTE_TEXT_GENERATOR_MODEL_CODE,
             }
         else:
```

Comment:
Don't just call else for all other inputs.

if:
elif:
else:
  raise ValueError

Reply:
Done